modules/howtos/pages/analytics-using-sdk.adoc

```diff
@@ -1,5 +1,5 @@
 = Analytics using the SDK
-:description: Parallel data management for complex queries over many records, using a familiar N1QL-like syntax.
+:description: Parallel data management for complex queries over many records, using a familiar {sqlpp}-like syntax.
```

Comment:
See also the comment above, but probably best simply as 
"using a familiar {sqlpp} syntax."